ui/components/config/meta-config-renderer.tsx

```
@@ -1,165 +1,165 @@
+                  Use <code className="rounded bg-neutral-100 px-1 py-0.5 text-neutral-800">env.&lt;VAR&gt;</code> to read the value from an
+                  environment variable.
+                </p>
+              </TooltipContent>
+            </Tooltip>
+          </TooltipProvider>
+        </CardTitle>
+      </CardHeader>
+      <CardContent className="space-y-4 px-0">
+        {config.fields.map((field) => (
+          <div key={field.name}>
+            <label className="block text-sm font-medium">{field.label}</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)